lib/teiserver_web/controllers/api/admin/user_controller.ex

```diff
@@ -0,0 +1,102 @@
+defmodule TeiserverWeb.API.Admin.UserController do
+  use TeiserverWeb, :controller
+  alias Teiserver.{Account, OAuth}
+
+  plug Teiserver.OAuth.Plug.EnsureAuthenticated, scopes: ["admin.user"]
+
+  @stat_fields ["mu", "sigma", "play_time", "spec_time", "lobby_time"]
+
+  # POST /teiserver/api/admin/users
```

Comment:
Not the biggest fan of these comments, as they are bound to fall out of sync. Usually, it's easy enough to grep the module name in `router.ex`, or run `mix phx.routes` to have this info.